application/classes/Ushahidi/Listener/ContactListener.php

```diff
@@ -60,6 +60,19 @@ public function setTargetedSurveyStateRepo(TargetedSurveyStateRepository $repo)
 		$this->targeted_survey_state = $repo;
 	}
 
+	// CreateRepository
+	private function getContactPostTitle($form_name, $contact_id)
+	{
+		try {
+			$uuid = substr(Ramsey\Uuid\Uuid::uuid4()->toString(), 0 , 5);
```

Comment:
@rjmackay we need to generate a random number that's around 4-6 characters long to use in the post title for posts that belong to a contact (just so they don't all have the same title). Ideas on a nicer way to do that?

Reply:
http://php.net/manual/en/function.random-int.php <- requires PHP 7 though.
But there's a userland implementation too https://github.com/paragonie/random_compat (we might already be using that I think)

Reply:
PHP has finally passed the bar of simple and safe crypto methods... its like a real programming language 😆

Reply:
ha! thanks I didn't realize that was a thing